### PR TITLE
RFC: remove references to common

### PR DESCRIPTION
**Pull request by sorja**
# Request for comments [RFC]
- Remove unused files
- Move _used_ function to scoped usage (ex. server/utils)
- Remove references to @common in server and client
- Add UserRoles.noRole "util"

### CODE REVIEW

File: client/pages/AssessmentSection/DataTable/GenerateValues/method.ts [hunk 1]
```diff
@@ -13,29 +7,29 @@ export enum Method {
...
-      })
-    }
-    // Linear or Repeat last require a minimum number of ODP
-    const noOdpsRequired = method === Method.linear ? 2 : 1
-    const odps: any = FraUtils.getOdps(data)
-    return odps.length >= noOdpsRequired
-  },
-}
+//
+// export const Methods = {
```
Comment (minotogna) [inline client/pages/AssessmentSection/DataTable/GenerateValues/method.ts:11]: why commented out?

Comment (sorja) [reply client/pages/AssessmentSection/DataTable/GenerateValues/method.ts:11]: Not used, (todo) see: https://github.com/openforis/fra-platform/pull/1477/files#diff-7dcd6666582341b145eb073ab080acf381031134866fd08b83e9137d565ec837L96

**Review by rfontanarosa — COMMENTED** [review]
you have removed common/numberFormat.ts but it's still imported by common/aggregate.ts

File: client/components/ThousandSeparatedDecimalInput/ThousandSeparatedDecimalInput.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
-import React from 'react'
-import * as R from 'ramda'
 import './ThousandSeparatedDecimalInput.scss'
-import { formatDecimal } from '@common/numberFormat'
+import React from 'react'
+
+import { Numbers } from '@core/utils'
 import { acceptableAsDecimal } from '@webapp/utils/numberInput'
+import * as R from 'ramda'
```
Comment (rfontanarosa) [inline client/components/ThousandSeparatedDecimalInput/ThousandSeparatedDecimalInput.tsx:6]: do we still use R? maybe we can wipe it out in another refactor...

Comment (sorja) [reply client/components/ThousandSeparatedDecimalInput/ThousandSeparatedDecimalInput.tsx:6]: We have this:
https://github.com/openforis/fra-platform/projects/1#card-76384309

**Review by minotogna — APPROVED** [review]

**Pull request merged**